src/components/Filters/CompanyReceivedFilter.jsx

```diff
@@ -0,0 +1,205 @@
+/* eslint complexity: ["error", 7] */
+import './DateFilter.less'
+import {
+    DATE_VALIDATION_FORMAT,
+    maxDate,
+    minDate
+} from '../../constants'
+import React, { useEffect, useMemo, useRef, useState } from 'react'
+import {
+    selectQueryCompanyReceivedMax,
+    selectQueryCompanyReceivedMin
+} from '../../reducers/query/selectors';
+import { useDispatch, useSelector } from 'react-redux'
+import { changeDates } from '../../actions/filter'
+import CollapsibleFilter from './CollapsibleFilter'
+import dayjs from 'dayjs'
+import dayjsCustomParseFormat from 'dayjs/plugin/customParseFormat'
+import dayjsIsBetween from 'dayjs/plugin/isBetween'
+import { formatDate } from '../../utils/formatDate';
+import iconMap from '../iconMap'
+
+dayjs.extend( dayjsCustomParseFormat )
+dayjs.extend( dayjsIsBetween )
```

Comment:
Missing `;`s 

We should update the linter to autofix these.

Reply:
agreed. 

We should toss this whole configuration and reformat using what we have set up in explorer with eslint, prettier, stylelint, etc. I think it works pretty well!